primitives/comb.fil

```diff
@@ -177,4 +178,42 @@ extern "comb.sv" {
    ) -> (
       out: ['G, 'L] OUT_WIDTH
    ) where 'L > 'G, IN_WIDTH > 0, OUT_WIDTH > 0;
+}
+
+// Reduces a bundle of N wires of size W to a single wire of size W*N
+// {010, 101, 111} -> 010101111
+comp ConcatBundle[W, N]<'G: 1>(
```

Comment:
Let's put these in a different file. In general, I want primitives files that expose externs to only contain the externs defined in the Verilog file.